### ISSUE TITLE
if constexpr is not a thing!

### ISSUE THREAD

**Issue by malb** [fplll/g6k#64]
This comment in `fht_lsh.cpp` is true for C++17 but not for C++11

```
    // But we don't want to use a regular if! That's slow.
    // Thankfully, if constexpr is a thing - it will throw away the branch that we don't care about in our template - 
    // thus it lets us write simpler code, but get the benefits we want in the end. 
```

This was already merged an now master fails.

**Comment by joerowell** [comment]
Let me fix it - we can get away with cpp17constexprif (my fault!)

**Comment by cr-marcstevens** [comment]
Just remove the constexpr in the if.
Code optimization should be the same.

**Comment by malb** [comment]
I'm annoyed at our CI, it should have caught this pre-merge.

**Comment by joerowell** [comment]
Does the CI force C++11?

**Comment by malb** [comment]
It does `-std=c++11` but that seems permissible. Maybe adding `-pedantic` would help?

**Comment by joerowell** [comment]
Closed via #65
